GOVERNANCE.md

```diff
@@ -3,43 +3,58 @@ the Benevolent dictator governance model by Ross
+Specific advice for contributing to the project can be found in
+[CONTRIBUTING.md](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md).
 
-- opening issues to report bugs or suggest new features
-- submitting PRs to implement new features, fix bugs, or improve documentation
-- commenting in open PRs or issues to help support users or contribute to discussion
+## Core developers
 
-## Users
-Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements.
+Core developers are collectively responsible for day-to-day development of the package, including responding to issues and reviewing pull requests. They are GitHub administrators and [package authors](https://github.com/tidyverse/ggplot2/blob/master/DESCRIPTION#L8), which means that they have the ability to make changes to project code, and receive credit when others cite the package.
 
-Users are encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+While core developers can modify code directly, this ability is rarely used. Instead, changes are proposed as pull requests, and are only merged after they have been reviewed by at least one other core developer. Changes to the API (especially breaking changes) must also be approved by the project lead. 
 
-- evangelising about the project
-- informing developers of project strengths and weaknesses from a new user’s perspective
-- providing moral support (a ‘thank you’ goes a long way)
+Core developers are recruited from contributors. An invitation to join the core team can be extended to anyone who has made a major contribution, either through a small number of large changes, or a consistent pattern of smaller contributions. Any existing core developer can propose a contributor be invited to the core team by emailing the project lead. The project lead will the confirm the invitation with existing core developers.
 
-Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described above.
+The core developers of ggplot2 are:
+
+* [Winston Chang](https://github.com/wch)
+* [Lionel Henry](https://github.com/lionel-)
+* [Thomas Lin Pedersen](https://github.com/thomasp85)
+* [Kohske Takahashi](https://github.com/kohske)
+* [Claus Wilke](https://github.com/clauswilke)
+* [Kara Woo](https://github.com/karawoo)
+
+All core developers are bound by the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Project lead
+
+The project lead, [Hadley Wickham](http://github.com/hadley), is responsible for:
+
+* Setting, and clearly communication the strategic objectives of the project.
```

Comment:
I defer to the project lead (obvi), but since these bullets aren't actually sentences, I'd do semicolons (or commas) and an "and" before the last one.